internal/config/fetch_test.go

```diff
@@ -1,75 +1,73 @@
 package config
 
-import (
-	"testing"
-)
+// TODO: this test is flaky. Should be revised
```

Comment:
please, make sure this test will be revised in some other task

Reply:
Ok, i've created a ticket in our issue tracker